features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -160,6 +162,16 @@ data class SessionReplayConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Defines the minimum system requirements for enabling the Session Replay feature.
+         * When [SessionReplay.enable] is invoked, the system configuration is verified against these requirements.
+         * If the system meets the specified criteria, Session Replay will be successfully enabled.
+         */
+        fun setSystemRequirement(systemRequirementConfiguration: SystemRequirementConfiguration): Builder {
```

Comment:
should it be `setSystemRequirements` (because there is at least two: CPU and memory)